src/pages/publication/FilesAndLicensePanel.tsx

```diff
@@ -61,7 +61,10 @@ const FilesAndLicensePanel: React.FC<FilesAndLicensePanelProps> = ({ goToNextTab
                     <FileCard
                       key={file.id}
                       file={file}
-                      removeFile={() => remove(i)}
+                      removeFile={() => {
+                        uppy.removeFile(file.id);
+                        remove(i);
```

Comment:
hvis 'i' står for index - kunne du skrevet det isteden ?

Reply:
i for index ja. Jeg syns det er såppas vanlig at i er nok. any others?

Reply:
faktisk bare i denne filen at det finnes kun `i`, ellers er det skrevet ut index

Reply:
javel daaaa